### DISCUSSION TITLE
Could not find a version that satisfies the requirement pywin32==223

### DISCUSSION THREAD

**Discussion by rahul6612** [mhammond/pywin32#2519] [General]
while pushing my project on heroku i am getting this error.
```
 Could not find a version that satisfies the requirement pywin32==223
 No matching distribution found for pywin32==223
```
any idea please tell me.

**Comment by Avasam** [comment]
@rahul6612  Do you have more logs? Is your host/CI/whatever a UNIX platform by any chance ? What flavor of Python are you using? (Anaconda, CPython, Cython, etc.). You should try again with `>= 228` for Python 3.8 support. (3.7 being EOL)

**Comment by Avasam** [comment]
https://www.heroku.com/dynos

> The [Heroku Platform](https://www.heroku.com/platform) uses the container model to run and scale all Heroku apps. The containers used at Heroku are called “dynos.” Dynos are isolated, virtualized Linux containers that are designed to execute code based on a user-specified command.
